File: algorithms/sorting/selection_sort/selection_sort.js
function selectionSort(arr) {
    for (let i = 0; i < arr.length; i++) {
        let lowestAtIndex = i;

        for (let j = i+1; j < arr.length; j++) {
            if (arr[j] < arr[lowestAtIndex]) {
                lowestAtIndex = j;
            }
        }

        if (lowestAtIndex !== i) {
            let temp = arr[i];
            arr[i] = arr[lowestAtIndex];
            arr[lowestAtIndex] = temp;
        }
    }
    return arr;
}

File: algorithms/sorting/selection_sort/selection_sort.py
def selection_sotr(arr):
    for i in range(len(arr)):
        lowest_at_index = i
        for j in range(i, len(arr)):
            if arr[j] < arr[lowest_at_index]:
                lowest_at_index = j
        if lowest_at_index != i:
            arr[i], arr[lowest_at_index] = arr[lowest_at_index], arr[i]
    
    return arr
        